### DISCUSSION TITLE
Why is nokmem ignored in runc?

### DISCUSSION THREAD

**Discussion by ningziwen** [opencontainers/runc#3688] [Q&A] [answered]
Hi I'm new to runc and trying to learn runc from reading the docs and source code.

I found in README it mentioned nokmem is ignored since 1.0.0-rc94. 

I searched nokmem and found the flag is to disable kernel memory. My understanding is this enabling this flag can prevent the app using kernel memory. As runc ignored this flag, that means runc allows using kernel memory.

Correct me if my understanding is wrong. If it is right, I'm curious what is was reason of making this change. What are the benefits did this change bring and what are the concerns of not having this change initially?

Thanks!

**Comment by AkihiroSuda** [comment, marked as answer]
Because Linux kernel removed support for it
https://github.com/torvalds/linux/commit/58056f77502f3567b760c9a8fc8d2e9081515b2d
